exonum-java-binding-core/rust/src/utils/errors.rs

```diff
@@ -3,11 +3,9 @@ use jni::JNIEnv;
 
 use std::cell::Cell;
 
-use utils::{get_class_name, get_exception_message};
+use utils::{get_class_name, get_exception_message, jni_cache};
```

Comment:
What about importing `get_error_class` here?

Reply:
I explained my general idea of imports and methods names below in comments.